src/liballoc/string.rs

```diff
@@ -128,8 +127,8 @@ use crate::vec::Vec;
 /// The [`bytes`] and [`chars`] methods return iterators over the first
 /// two, respectively.
 ///
-/// [`bytes`]: #method.bytes
-/// [`chars`]: #method.chars
+/// [`bytes`]: str::bytes
+/// [`chars`]: str::chars
```

Comment:
In alloc, this points to https://doc.rust-lang.org/nightly/std/primitive.str.html#method.bytes .
But in std, this points to absolute local link: http://0.0.0.0:8000/home/lzutao/fork/rust/compiler/src/liballoc/std/primitive.str.html#method.bytes

And yes, I run http server in doc directory.

Reply:
Please file an issue for this bug.

Reply:
Actually this is related to cross crate re-exports. So I bet the behavior will change when #73101 lands.

Reply:
After stage1, the location in std is Ok http://0.0.0.0:8000/std/primitive.str.html#method.bytes .